cli/src/commands/rebase.rs

```diff
@@ -618,3 +565,38 @@ fn check_rebase_destinations(
     }
     Ok(())
 }
+
+/// Print details about the provided [`MoveCommitsStats`].
+fn print_move_commits_stats(ui: &Ui, stats: MoveCommitsStats) -> std::io::Result<()> {
```

Comment:
nit: pass `&MoveCommitsStats` by reference (and destructure as `let &MoveCommitsStatus ..`